x/ccv/consumer/keeper/relay.go

```diff
@@ -25,7 +24,12 @@ import (
 //
 // Note: CCV uses an ordered IBC channel, meaning VSC packet changes will be accumulated (and later
 // processed by ApplyCCValidatorChanges) s.t. more recent val power changes overwrite older ones.
-func (k Keeper) OnRecvVSCPacket(ctx sdk.Context, packet channeltypes.Packet, newChanges ccv.ValidatorSetChangePacketData) exported.Acknowledgement {
```

Comment:
Are you sure about this? You're changing the signature of an ibc module method?

Reply:
This is the signature of a keeper function called by an IBC module method. The IBC logic is in i`bc_module.go` and I would like as much as possible to keep it there. See https://github.com/cosmos/ibc-go/blob/9db308b8fe1713f8d503b3e54996fc326fad28be/modules/apps/transfer/ibc_module.go#L193